ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/EventsTableInsert.scala

```diff
@@ -159,17 +158,17 @@ private[events] trait EventsTableInsert { this: EventsTable =>
     * @throws RuntimeException If a value cannot be serialized into an array of bytes
     */
   @throws[RuntimeException]
-  def prepareBatchInsert[Nid <: NodeId](
+  def prepareBatchInsert(
```

Comment:
I believe @S11001001 had a good idea why this was here, but I don't know what it was.